tiered-storage/jcloud/src/main/java/org/apache/bookkeeper/mledger/offload/jcloud/JCloudLedgerOffloaderFactory.java

```diff
@@ -38,13 +40,14 @@ public static JCloudLedgerOffloaderFactory of() {
 
     @Override
     public boolean isDriverSupported(String driverName) {
-        return BlobStoreManagedLedgerOffloader.driverSupported(driverName);
+        return JCloudBlobStoreProvider.driverSupported(driverName);
     }
 
     @Override
-    public BlobStoreManagedLedgerOffloader create(OffloadPolicies offloadPolicies,
-                                                  Map<String, String> userMetadata,
-                                                  OrderedScheduler scheduler) throws IOException  {
-        return BlobStoreManagedLedgerOffloader.create(offloadPolicies, userMetadata, scheduler);
+    public BlobStoreManagedLedgerOffloader create(OffloadPolicies offloadPolicies, Map<String, String> userMetadata,
+            OrderedScheduler scheduler) throws IOException {
+        
+        TieredStorageConfiguration config = TieredStorageConfiguration.create(userMetadata);
+        return BlobStoreManagedLedgerOffloader.create(config, userMetadata, scheduler);
```

Comment:
Currently, the Pulsar support independent offload policy for each namespace, the `userMetadata` param contains the global offload configs and the `offloadPolicies` param contains the Namespace level offload configs. If the namespace has its own offload policy, we should use the independent offload policy, otherwise, we should use the global offload policy. 

The `OffloadPolicies` has independent config for S3 and GCS, such as `s3ManagedLedgerOffloadRegion` and `gcsManagedLedgerOffloadRegion`, follow your way, we could combine them as the `region`.

Reply:
The focus of this PR is to refactor the tiered storage to use JCloud. I feel that we need to keep this PR to just that. I suggest we create an issue for this and assign it to me. I will make the change after this PR has been successfully merged.

Reply:
Sorry for the late response. If we don't care about the OffloadPolicies config, it will cause the namespace offload policies invalid.